debian-repo/index.html

```diff
@@ -106,7 +106,7 @@ <h2>Package installation</h2>
             <p>Install packages directly from the repository:</p>
 
             <pre><code>## Install a package
-sudo apt install ollama</code></pre>
+sudo apt install ollama libollama-[amd|nvidia]</code> (choose the package that fits your gpu!)</pre>
```

Comment:
Actually, `apt`'s dependency resolution picks the first listed if OR (|) is used in a package `Depends` (or by extension, `Recommends`).  See [here](https://www.debian.org/doc/manuals/aptitude/ch02s03s02.en.html).
```suggestion
sudo apt install ollama</code></pre>

            <div class="note">
                <strong>Note:</strong>The first GPU package that this <pre>ollama</pre> package recommends is <pre>libollama-nvidia</pre>, and so if none are requested, this will be installed by default.  If you don't need any, remove it afterwards, or add <pre>--no-install-recommends</pre> to your <pre>apt</pre> command.
            </div>
```

I'm going to move the other HTML note up higher.